tests/console/wicked_before_test.pm

```diff
@@ -13,13 +13,13 @@
 use base "consoletest";
 use strict;
 use testapi;
-use utils qw(systemctl setup_static_network);
+use utils;
```

Comment:
Why? using *qw(systemctl setup_static_network)* should be more efficient. @foursixnine knows.

Reply:
No, I removed that sleep call with e0aea3ce already :)

If you find any more then please
* remove them
* don't accept PRs adding any

It's not (only) about performance but about knowing explicitly what functions are used from where

Reply:
In any case automation test should not care about each millisecond for sure . It is not unit tests , our tests approximate run is  30 minutes  or even more. Why we should care about few milliseconds which no one measure BTW ?
Can you clarify use case   for " knowing explicitly what functions are used from where" ? I mean if function is already mentioned in file you can just use it , if not you anyway need search globally to find a file where it is declared. So how you can use info that is_sle is declared in utils ?